packages/orchestrator/lib/routes/v1/postImmediate.ts

```diff
@@ -110,7 +110,7 @@ const handler = (scheduler: Scheduler) => {
             res.status(500).json({ error: { code: 'immediate_failed', message: task.error.message } });
             return;
         }
-        res.status(200).json({ taskId: task.value.id, retryKey: task.value.retryKey });
+        res.status(200).json({ taskId: task.value.id, retryKey: task.value.retryKey! });
```

Comment:
a freshly created task always has a retryKey. It is only null for task created before the introduction of retryKey